tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "pretty": true,
     "removeComments": false,
     "sourceMap": true,
+    "strictNullChecks": true,
```

Comment:
I added this do to the [recommended settings of material UI](https://github.com/regen-network/regen-sanity/pull/14) to try and figure out a bug - I don't think it was related, but leaving just because it's recommended